src/images/falseColourPanel.component.tsx

```diff
@@ -55,6 +63,111 @@ interface FalseColourPanelProps extends FalseColourParams {
   changeUpperLevel: (value: number | undefined) => void;
 }
 
+function MyListSubheader(props: ListSubheaderProps) {
+  return <ListSubheader {...props} />;
+}
+
+const ColourMapSelectMain = (
+  colourMap: string,
+  handleColourMapChange: (event: SelectChangeEvent) => void,
+  mainColourMap: ColourMapsParams
+) => {
+  const mainColourMapTypeName = Object.keys(mainColourMap)[0];
+  const mainColourMapNames = Object.values(mainColourMap)[0];
+
+  return (
+    <Select
+      labelId="colour-map-select-label"
+      id="colour-map-select"
+      value={colourMap}
+      label="Colour Map"
+      onChange={handleColourMapChange}
+    >
+      <MenuItem value="">
+        <em>Default</em>
+      </MenuItem>
+
+      <MyListSubheader>{mainColourMapTypeName}</MyListSubheader>
+      {mainColourMapNames.map((colourMap) => (
+        <MenuItem key={colourMap} value={colourMap}>
+          {colourMap}
+        </MenuItem>
+      ))}
+    </Select>
+  );
+};
+
+const ColourMapSelectExtended = (
+  colourMap: string,
+  handleColourMapChange: (event: SelectChangeEvent) => void,
+  extendedColourMapNames: ColourMapsParams
+) => {
+  const mainColourMapTypeNames = Object.keys(extendedColourMapNames);
+  const mainColourMapNames = Object.values(extendedColourMapNames);
```

Comment:
These variables should probably be renamed to have the prefix `extended` rather than `main`